pkg/storage/unified/sql/sqltemplate/dialect_postgresql.go

```diff
@@ -35,3 +35,11 @@ func (p postgresql) Ident(s string) (string, error) {
 
 	return p.standardIdent.Ident(s)
 }
+
+func (postgresql) CurrentEpoch() string {
+	return "EXTRACT(EPOCH FROM clock_timestamp()) * 1000000::BIGINT"
```

Comment:
> transaction_timestamp() is equivalent to CURRENT_TIMESTAMP, but is named to clearly reflect what it returns. statement_timestamp() returns the start time of the current statement (more specifically, the time of receipt of the latest command message from the client). statement_timestamp() and transaction_timestamp() return the same value during the first command of a transaction, but might differ during subsequent commands. clock_timestamp() returns the actual current time, and therefore its value changes even within a single SQL command. timeofday() is a historical PostgreSQL function. Like clock_timestamp(), it returns the actual current time, but as a formatted text string rather than a timestamp with time zone value. now() is a traditional PostgreSQL equivalent to transaction_timestamp().

Reply:
IIUC, `clock_timestamp()` in Postgres is like `SYSDATE()` in MySQL, and `statement_timestamp()` in Postgres is like `NOW()` in MySQL.
For example, I get this in Posgres:
```
postgres=# select EXTRACT(EPOCH FROM clock_timestamp()) * 1000000::BIGINT, EXTRACT(EPOCH FROM clock_timestamp()) * 1000000::BIGINT;
        ?column?         |        ?column?         
-------------------------+-------------------------
 1727397294380888.000000 | 1727397294380896.000000
(1 row)

postgres=# select EXTRACT(EPOCH FROM statement_timestamp()) * 1000000::BIGINT, EXTRACT(EPOCH FROM statement_timestamp()) * 1000000::BIGINT;
        ?column?         |        ?column?         
-------------------------+-------------------------
 1727397304674593.000000 | 1727397304674593.000000
(1 row)
```
And in MySQL:
```
mysql> SELECT CAST(UNIX_TIMESTAMP(SYSDATE(6)) * 1000000 AS SIGNED), CAST(UNIX_TIMESTAMP(SYSDATE(6)) * 1000000 AS SIGNED);
+------------------------------------------------------+------------------------------------------------------+
| CAST(UNIX_TIMESTAMP(SYSDATE(6)) * 1000000 AS SIGNED) | CAST(UNIX_TIMESTAMP(SYSDATE(6)) * 1000000 AS SIGNED) |
+------------------------------------------------------+------------------------------------------------------+
|                                     1727397411974346 |                                     1727397411974364 |
+------------------------------------------------------+------------------------------------------------------+
1 row in set (0.00 sec)

mysql> SELECT CAST(UNIX_TIMESTAMP(NOW(6)) * 1000000 AS SIGNED), CAST(UNIX_TIMESTAMP(NOW(6)) * 1000000 AS SIGNED);
+--------------------------------------------------+--------------------------------------------------+
| CAST(UNIX_TIMESTAMP(NOW(6)) * 1000000 AS SIGNED) | CAST(UNIX_TIMESTAMP(NOW(6)) * 1000000 AS SIGNED) |
+--------------------------------------------------+--------------------------------------------------+
|                                 1727397413952729 |                                 1727397413952729 |
+--------------------------------------------------+--------------------------------------------------+
1 row in set (0.00 sec)
```

But honestly, the difference is so subtle that most probably nothing will change in practice, so marking this as resolved, feel free to not apply it.